.github/workflows/eleventy_build.yml

```diff
@@ -20,6 +20,8 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./docs
           publish_branch: deploy_production
+        env:
+          SITE_DOMAIN: "https://staging.drought.ca.gov"
```

Comment:
Don't have this subdomain setup yet but going to request it.

I don't know if this is the best way to solve this but we can setup more environments here https://github.com/cagov/drought.ca.gov/settings/environments, then define values that get pulled in to these build files as variables with the ${{ }} syntax

I made you admin on drought.ca.gov

Reply:
cool thank you